client/src/core/client/stream/tabs/Comments/Comment/MediaConfirmation/MediaConfirmationIcon.tsx

```diff
@@ -20,6 +21,9 @@ const MediaConfirmationIcon: FunctionComponent<Props> = ({ media }) => {
       {media.type === "twitter" && (
         <SvgIcon size="xs" filled="currentColor" Icon={XLogoTwitterIcon} />
       )}
+      {media.type === "bsky" && (
```

Comment:
for some reason, I'm not seeing the bluesky logo icon with the media confirmation prompt?

Reply:
this has been fixed

Reply:
have also updated with the streamline icon now

Reply:
thank you!